Core/Security/m2crypto/X509Certificate.py

```diff
@@ -0,0 +1,478 @@
+""" X509Certificate is a class for managing X509 certificates
+
+Proxy RFC: https://tools.ietf.org/html/rfc38200
+
+X509RFC: https://tools.ietf.org/html/rfc5280
+
+"""
+
+__RCSID__ = "$Id$"
+
+import datetime
+import os
+import random
+import time
+
+import M2Crypto
+
+
+from DIRAC import S_OK, S_ERROR
+from DIRAC.Core.Utilities import DErrno
+from DIRAC.ConfigurationSystem.Client.Helpers import Registry
+from DIRAC.Core.Security.m2crypto import asn1_utils
+from DIRAC.Core.Utilities.Decorators import executeOnlyIf
+
+
+# Init the rand seed
+random.seed()
```

Comment:
I'm not sure this call is really required? Python should just seed by time on the first random call anyway...

Reply:
Correct, I'll remove it